README.md

```diff
@@ -17,30 +17,30 @@ The client libraries are supported on Python 2.7 and 3.5.3 or later.
 
 ## Packages available
 Each service might have a number of libraries available from each of the following categories:
-* [Client - July 2019 Preview](#Client-July-2019-Preview)
-* [Client - Stable](#Client-Stable)
+* [Client - November 2019 Releases](#Client-November-2019-Releases)
+* [Client - Previous Versions](#Client-Previous-Versions)
 * [Management](#Management)
 
 
-### Client: July 2019 Preview
-New wave of packages that we are currently releasing in **Preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azuresdkspecs.z5.web.core.windows.net/PythonSpec.html).
+### Client: November 2019 Releases
+New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azure.github.io/azure-sdk/python_introduction.html).
```

Comment:
I'm good with merging this, but the term "GA" is a very Microsoft:y term. You generally have pre-release and stable releases.

Reply:
100% agreed, will have another wave of changes soon - for now just want to make sure we actually point to releases